### DISCUSSION TITLE
Adapt this for Gnosis Chain?

### DISCUSSION THREAD

**Discussion by biafra23** [serenita-org/ethstaker.tax#54] [Q&A]
How involved do you think it would be to adapt this for Gnosis Chain? Can I just point it to a Gnosis beacon chain Lighthouse client?

**Comment by eth2353** [comment]
I don't think it would require too many changes, but it would definitely require _some_ changes. Some examples:

- `SLOT_TIME` and `SLOTS_PER_EPOCH` differ for Gnosis Chain - this would very likely cause issues somewhere
- the frontend has "ETH" hardcoded as the reward currency
- there may be parts in the code that rely on the initial deposit to have a value of 32

If you'd like to try it out / work on it, I'd be happy to merge it if the required changes can be kept at a higher level (env variables / parameters).
